development/contracts/WhitelistArbitrator.sol

```diff
@@ -108,9 +108,8 @@ contract WhitelistArbitrator is BalanceHolder {
     }
 	
     /// @notice Return the dispute fee for the specified question. 0 indicates that we won't arbitrate it.
-    /// @param question_id The question in question
     /// @dev Uses a general default, but can be over-ridden on a question-by-question basis.
-    function getDisputeFee(bytes32 question_id)
+    function getDisputeFee()
```

Comment:
This should keep this parameter as it's in the API

Reply:
You are saying that this whitelist arbitrator contract is also inheriting some API?
It should inherit IArbitrator, right? I will do that in another PR then.

Reply:
I can not inherit IArbitrator, as some function are not implemented or different :) I leave that for another PR.